src/pcapi/routes/native/v1/serialization/account.py

```diff
@@ -135,12 +138,19 @@ def _get_booked_offers(user: User) -> dict:
 
         return {booking.stock.offer.id: booking.id for booking in not_cancelled_bookings}
 
+    @staticmethod
+    def _get_needs_to_validate_phone(user: User) -> bool:
+        if feature_queries.is_active(FeatureToggle.ENABLE_PHONE_VALIDATION):
+            return user.is_eligible and not user.isBeneficiary and not user.is_phone_validated
+        return False
```

Comment:
(détail / style)

```suggestion
        if not feature_queries.is_active(FeatureToggle.ENABLE_PHONE_VALIDATION):
            return False
        return user.is_eligible and not user.isBeneficiary and not user.is_phone_validated
```

1. La lecture est un peu plus fluide : on traite le cas particulier (_feature flag_ désactivé) en premier.
2. Si un jour on supprime le feature flag (je ne sais pas si c'est prévu), le diff sera plus facile à lire.